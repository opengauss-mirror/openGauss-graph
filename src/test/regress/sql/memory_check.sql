\c postgres
select * from pg_sleep(10);
select  case  when memorymbytes > 1720 then 'fail' else 'pass' end as result from gs_total_memory_detail where memorytype = 'process_used_memory';