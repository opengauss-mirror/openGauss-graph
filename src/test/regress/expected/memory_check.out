\c postgres
select * from pg_sleep(10);
 pg_sleep 
----------
 
(1 row)

select  case  when memorymbytes > 1720 then 'fail' else 'pass' end as result from gs_total_memory_detail where memorytype = 'process_used_memory';
 result 
--------
 pass
(1 row)

